jsonpb/jsonpb.go

```diff
@@ -494,6 +494,24 @@ func (m *Marshaler) marshalValue(out *errWriter, prop *proto.Properties, v refle
 		return out.err
 	}
 
+	// Handle special cases for float, double.
```

Comment:
Sorry, another nit.  Please update comment to be more specific, perhaps ...

// Handle non-finite floats like NaN, Infinity and -Infinity.